ruby-challenges.rb

```diff
@@ -32,3 +45,15 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def is_palindrome (word)
+    if(word == word.reverse.capitalize)
```

Comment:
Capitalize works great for these test variables, however, what happens if the word is all lowercase?